packages/components/package.json

```diff
@@ -48,9 +51,9 @@
     "ember-auto-import": "^2.6.3",
     "ember-cached-decorator-polyfill": "^0.1.4",
     "ember-cli-babel": "^7.26.11",
-    "ember-cli-clipboard": "^1.0.0",
     "ember-cli-htmlbars": "^6.2.0",
     "ember-cli-sass": "^10.0.1",
+    "ember-cli-typescript": "^5.2.1",
```

Comment:
this package is entering maintainece mode and is no longer the recommended way to use TS https://github.com/typed-ember/ember-cli-typescript/issues/1568